content/blog/eng/v-protetto14-4-2023.md

```diff
@@ -0,0 +1,93 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-18"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto 14/4/2023"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+# Venerdì Protetto 2023-04-14
+
+This page contains the abstracts of the talks held during the Venerdì Protetto on April 14, 2023. 
```

Comment:
I woul write a little more here, since this first paragraph build the post preview on the homepage and list pages. Maybe even using an automatic TOC may be enough?

Reply:
I added the speakers and topics section to extend the preview and provide useful info when browsing the page. 
I believe that a TOC wouldn't show up in the preview.